vm/src/obj/objfunction.rs

```diff
@@ -83,6 +95,28 @@ fn member_get(vm: &mut VirtualMachine, mut args: PyFuncArgs) -> PyResult {
     }
 }
 
+fn data_get(vm: &mut VirtualMachine, mut args: PyFuncArgs) -> PyResult {
+    match args.shift().get_attr("fget") {
+        Some(function) => vm.invoke(function, args),
+        None => {
+            println!("A");
```

Comment:
Were this line and the `println!("B")` below left in by mistake?

Reply:
Yes :frowning_face: